ci/env.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# values can also be set for env vars DOCKER_PASSWORD, DOCKER_USERNAME and DOCKER_REGISTRY
```

Comment:
You can set the following environment variables to specify your docker credentials or docker registry